client/tests/acceptance/user-can-click-package-edit-test.js

```diff
@@ -34,12 +35,18 @@ module('Acceptance | user can click package edit', function(hooks) {
     await visit('/projects');
     await click('[data-test-project="edit-pas"]');
     await click('[data-test-save-button]');
+
+    await waitFor('[data-test-submit-button]:not([disabled])');
     await click('[data-test-submit-button]');
     await click('[data-test-confirm-submit-button]');
 
     // for some reason promises aren't being captured so we await for settled state
     await settled();
 
+    // use waitFor as a way to "wait" for the transition
+    // within the pas-form/edit Component submit() task
+    await waitFor('[data-test-show-dcprevisedprojectname]');
```

Comment:
Wanna flag this for @trbmcginnis and @bfreeds and @andycochran — these two methods (awaiting settled and waitFor) are some good tools for your testing toolbox. the # 1 prob you will find with weird testing issues is dealing with asynchrony — teh test assertion not realizing there are promises/tasks still resolving. these helpers will assist with that.